packages/runner-ct/package.json

```diff
@@ -16,23 +16,39 @@
     "watch": "webpack --watch --progress --config webpack.config.ts"
   },
   "dependencies": {
+    "debug": "^4.3.2",
+    "lodash": "^4.17.20",
+    "send": "^0.17.1",
+    "tslib": "^2.3.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.3",
+    "@babel/preset-env": "^7.12.1",
     "@cypress/design-system": "0.0.0-development",
     "@cypress/react-tooltip": "0.5.3",
     "@fortawesome/free-regular-svg-icons": "5.15.2",
     "@fortawesome/react-fontawesome": "^0.1.14",
+    "@packages/driver": "0.0.0-development",
+    "@types/node": "14.14.31",
+    "@types/sockjs-client": "1.1.0",
     "ansi-to-html": "0.6.14",
+    "babel-loader": "8.1.0",
     "bluebird": "3.5.3",
     "cash-dom": "^8.1.0",
     "chai": "^4.2.0",
     "classnames": "2.3.1",
+    "clean-webpack-plugin": "^3.0.0",
     "cypress-real-events": "1.4.0",
+    "dart-sass": "^1.25.0",
+    "eslint-plugin-mocha": "^8.0.0",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
     "express": "^4.17.1",
     "fuzzysort": "^1.1.4",
     "globby": "^11.0.1",
     "hotkeys-js": "3.8.2",
     "html-webpack-plugin": "^4.5.0",
     "koa": "^2.13.0",
-    "lodash": "^4.17.20",
     "mobx": "5.15.4",
```

Comment:
`mobx` is called in `src/lib/state.ts`. It should stay in the dependencies.

Reply:
We only need server-run dependencies in the `dependencies`, the rest that are client side can be in `devDependencies`